docs/how/search.md

```diff
@@ -1,17 +1,76 @@
-# Search Guide
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
-## Introduction 
+# About DataHub Search
 
-The search bar is one of the means of finding data in Datahub. In this document, we discuss more effective ways of finding information beyond doing a standard keyword search. This is because keyword searches can return results from almost any part of an entity.
+<!-- All Feature Guides should begin with `About DataHub ` to improve SEO -->
 
-### Search in Specific Fields
+<!-- 
+Update feature availability; by default, feature availabilty is Self-Hosted and Managed DataHub
 
-The following examples are in the format of  
-X: *typical question* :  
+Add in `saasOnly` for Managed DataHub-only features
+ -->
+
+<FeatureAvailability/>
+
+The **search bar** is an important mechnaism for discovering data assets in DataHub. From the search bar, you can find Datasets, Columns, Dashboards, Charts, Data Pipelines, and more. Simply type in a term and press 'enter'. 
+
+![](https://github.com/datahub-project/static-assets/blob/main/imgs/search-landingpage.png?raw=true)
+
+**Advanced queries** and the **filter sidebar** helps fine tuning queries. For programmatic users Datahub provides a **GraphQL API** as well. 
+
+<!-- This section should provide a plain-language overview of feature. Consider the following:
+
+* What does this feature do? Why is it useful?
+* What are the typical use cases?
+* Who are the typical users?
+* In which DataHub Version did this become available? -->
+
+## Search Setup, Prerequisites, and Permissions
+
+<!-- This section should provide plain-language instructions on how to configure the feature:
+
+* What special configuration is required, if any?
+* How can you confirm you configured it correctly? What is the expected behavior?
+* What access levels/permissions are required within DataHub? -->
+Search is available for all users. Although Search works out of the box, the more relevant data you ingest, the better the results are.
```

Comment:
Let's elaborate on this - what is relevant data here? What types of metadata improve Search results?